components/ResourceImageLayout.jsx

```diff
@@ -25,7 +25,7 @@ import { baseTheme } from "../theme";
  * |- box shadow and rounded borders
  */
 
-export const StudentResource = ({
+export const ResourceImageLayout = ({
```

Comment:
Just renaming the component in `<component>.jsx` is not enough, it's is mentioned in 10 other places in the project. When you change the name, it needs to renamed in the other files as well (where this component is exported/imported)

Reply:
I also don't know that "ResourceImageLayout" is the best name, since the problem with "Resource" is that this isn't always used to show a PuMP resource (used in UAS 3 Step as their 3 steps). I'm not sure what's a great name either, since this is really similar to the `LeftImageTextLayout` but at least `ImageTextLayout` to describe what can be added (image, title, description, buttons) and the image position can be changed.

Reply:
Alright, would "SectionImageTextLayout" work?

Reply:
I don't think adding `Section` makes sense because`sections/` has components that consist of components from `components/` (section != component, a section would be like UAS3Step that uses 3 of the StudentResource components)